.env.example

```
@@ -10,6 +10,7 @@ POSTGRES_DB=postgres
 # Redis settings
 CELERY_BROKER_URL=redis://redis:6379/0
 CELERY_RESULT_BACKEND=redis://redis:6379/0
+CACHE_URL=redis://redis:6379/0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CACHE_URL key should go before the CELERY_BROKER_URL key

(UnorderedKey)